src/map/mob.hpp

```diff
@@ -398,6 +398,7 @@ struct mob_data {
 	uint16 damagetaken;
 
 	e_mob_bosstype get_bosstype();
+	map_session_data* get_mvp();
```

Comment:
```suggestion
	map_session_data* get_mvp_player();
```
Just to make it more obvious? Otherwise it may be misinterpreted as getting the MVP flag from the DB or something like this?